src/AppInstallerCLIE2ETests/AppInstallerCLIE2ETests.csproj

```diff
@@ -11,11 +11,17 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Msix.Utils" Version="2.0.3" />
     <PackageReference Include="Microsoft.Win32.Registry" Version="5.0.0" />
+    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.6.4" />
     <PackageReference Include="nunit" Version="3.12.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.15.1" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
   </ItemGroup>
 
+  <!-- CsWinRT properties -->
+  <PropertyGroup>
+    <CsWinRTWindowsMetadata>10.0.19041.0</CsWinRTWindowsMetadata>

```

Comment:
>10.0.19041.0 [](http://example.com/codeflow?start=28&length=12)

nit: 10.0.17763.0 is the min version winget supports, if possible, I think we prefer this version

Reply:
Updated to 10.0.17763.0 👍